gammapy/estimators/excess_map.py

```diff
@@ -33,18 +33,18 @@ def convolved_map_dataset_counts_statistics(dataset, kernel, mask):
         background = dataset.background * mask
         background.data[dataset.acceptance_off.data == 0] = 0.0
         n_off = dataset.counts_off * mask
-
-        background_conv = background.convolve(kernel.array)
-        n_off_conv = n_off.convolve(kernel.array)
-
         npred_sig = dataset.npred_signal() * mask
-        mu_sig = npred_sig.convolve(kernel.array)
+
+        background_convolve = background.convolve(kernel.array)
+        npred_sig_convolve = npred_sig.convolve(kernel.array)
+        if correlate_off:
+            n_off = n_off.convolve(kernel.array)
 
         with np.errstate(invalid="ignore", divide="ignore"):
```

Comment:
The `alpha` you want to use is either :
- `alpha_conv = background_conv / n_off_conv` if you want to apply correlation (i.e. it the off weighted means of the alpha).
- `alpha = acceptance_conv/acceptance_off` if you don't correlate the OFFs, since in that case the `acceptance_off` is left untouched.

Reply:
Ok. For the off correlation case, will it not be better to use also the correlated acceptance On and correlated acceptance Off ?

Reply:
A priori, we don't return the correlated `acceptance` and the correlated `acceptance_off`. So it might not be necessary to compute them, the point is to get the correct `alpha`.